### ISSUE TITLE
[java] UnusedImports false positive with Javadoc link

### ISSUE THREAD

**Issue by danjee** [pmd/pmd#936]
**Affects PMD Version:**  5.3.5 (m-pmd-p 3.6)

**Rule:** UnusedImports

**Description:** When using the javadoc `{@link package.name.ClassName}` a import is made on the class but the validation fails saying the import is not used

**Code Sample demonstrating the issue:**

```java
         /**
         * FIXME Also change this when doing change here
         * {@link ro.niinoo.CreateLoveOnEarth#init()}
         */

```

**Running PMD through:** *[Maven]*

**Comment by adangel** [comment]
Related previous bugs: #348 , #154 , [sf-1547](https://sourceforge.net/p/pmd/bugs/1547/) , [sf-1465](https://sourceforge.net/p/pmd/bugs/1465/)

**Comment by jsotuyod** [comment]
The non-typeresolution rule in old pmd versions may fail this way.

However, under current PMD versions this should be working properly.

Therefore I'm closing this.

**Comment by danjee** [comment]
I will double check. Thanks for quick hints.
